waspc/src/Wasp/Analyzer/Parser/Token.hs

```diff
@@ -26,6 +15,31 @@ data TokenType
   | TEOF
   deriving (Eq, Show)
 
+data SpecialChar
+  = LParen
+  | RParen
+  | LSquare
+  | RSquare
+  | LCurly
+  | RCurly
+  | Comma
+  | Colon
+  deriving (Eq, Show)
+
+data Keyword
+  = Import
+  | From
+  | WaspTrue -- avoids conflicts with Prelude.True and Prelude.False
+  | WaspFalse
+  deriving (Eq, Show, Enum, Bounded)
+
+prettyShowKeyword :: Keyword -> String
+prettyShowKeyword keyword = case keyword of
+  Import -> "import"
+  From -> "from"
+  WaspFalse -> "false"
+  WaspTrue -> "true"
```

Comment:
This allows us to enumerate/list all special characters and keywords (i.e., transfer them from the type space to the value space).

Reply:
Yeah, if we are doing all this grouping, it would also make sense to group literals.

Quoters -> Hm yes those are a bit tricky. Maybe best not to touch them. You could, in theory, put TLQuote and TRQuote into symbols, and TQuote is left standalone. But that doesn't feel great.